samples/ios-unified/AppDelegate.cs

```diff
@@ -14,6 +14,9 @@
 using System.Linq;
 using Xamarin.Forms;
 using System.Security.Authentication.ExtendedProtection;
+
+[assembly: Preserve (typeof (UALocation), AllMembers = true)]
```

Comment:
if this is required to make the locationProviderDelegate work properly, would it be simpler to just use UALocation directly? Using the delegate in the native sample is really more of a hack, based on the awkwardness of dependencies that would have resulted with AirshipDebugKit.

Reply:
Changed.